material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -85,7 +87,11 @@
   --md-mermaid-sequence-note-border-color: var(--md-mermaid-label-fg-color);
   --md-mermaid-sequence-number-bg-color: var(--md-mermaid-node-fg-color);
   --md-mermaid-sequence-number-fg-color: var(--md-accent-bg-color);
-}
+
+  /*custom code admonition colors*/
+
+  --md-code-admo-border-color: var(--code-green);
+  --md-code-admo-title-background: var(--code-green-light);
```

Comment:
I typically use `bg` for background and `fg` for foreground. So, I recommend changing this to `md-code-admo-title-bg` instead